app/src/lib/editors/win32.ts

```diff
@@ -18,6 +18,16 @@ interface IWindowsAppInformation {
 
 type RegistryKey = { key: HKEY; subKey: string }
 
+type WindowsExternalEditorJetbrainsToolbox = {
+  /**
+   * Default shell script name for JetBrains Products
+   * To get the script name go to:
+   * JetBrains Toolbox > Editor settings > Shell script name
+   * Note: Don't use on JetBrains community and edu editions
```

Comment:
Why  not? 🤔  Probably worth mentioning that

Reply:
The main issue is that JetBrains Toolbox generated the same script name for products without considering their edition.
JetBrains PyCharm Example:
1. Install JetBrains PyCharm Community
2. At this point, JetBrains Toolbox will generate a shell script called `pycharm`
3. Install JetBrains PyCharm Professional
4. JetBrains Toolbox will generate a new script with the same name, `pycharm` and will override the script generated for the community version

I added this example to [editor-integration.md](https://github.com/desktop/desktop/pull/15780/files#diff-2a91c8695284b34dee3845d11a7595422c8dd1a773c7b43e31cdd8a89655104c) and more documentation on how the auto-generation process works